tests/application/cms/test_views.py

```diff
@@ -799,3 +800,30 @@ def test_copy_measure_page(test_app_client, logged_in_dev_user):
 
     assert page.find("h1").text == "Edit page"
     assert page.find("input", attrs={"name": "title"})["value"] == "COPY OF Test Measure Page"
+
+
+def test_list_measure_versions_shows_expected_links(test_app_client, logged_in_dev_user):
+    measure_version_1_0 = MeasureVersionFactory(status="APPROVED", latest=False, version="1.0")
+    MeasureVersionFactory(status="APPROVED", latest=True, version="1.1", measure=measure_version_1_0.measure)
+
+    topic_slug = measure_version_1_0.measure.subtopic.topic.slug
+    subtopic_slug = measure_version_1_0.measure.subtopic.slug
+    measure_slug = measure_version_1_0.measure.slug
+
+    response = test_app_client.get(
+        url_for(
+            "cms.list_measure_versions", topic_slug=topic_slug, subtopic_slug=subtopic_slug, measure_slug=measure_slug
+        )
+    )
+
+    assert response.status_code == 200
+    page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
+
+    assert page.find("h1").text.strip().startswith("Version history")
```

Comment:
Maybe test the whole title, ie:

```python
assert page.find("h1").text.strip() == "Version history of Measure Name"
```

Reply:
The title is weirdly splintered with loads of spacing in between ... but I can add an assertion that the measure name is in there too, yeah.